README.md

```diff
@@ -42,14 +42,24 @@ Create a new Python file and add the following code, replacing the model identif
 ['https://replicate.com/api/models/stability-ai/stable-diffusion/files/50fcac81-865d-499e-81ac-49de0cb79264/out-0.png']
 ```
 
-Some models, particularly language models, may not require the version string. Refer to the API documentation for the model for more on the specifics:
+Some models, particularly language models, may not require the version string. You can always refer to the API documentation on the model page for specifics (for example, [check out the Llama 3 API documentation](https://replicate.com/meta/meta-llama-3-70b-instruct/api)).
 
 ```python
 replicate.run(
-    "meta/llama-2-70b-chat",
+    "meta/meta-llama-3-70b-instruct",
     input={
-        "prompt": "Can you write a poem about open source machine learning?",
-        "system_prompt": "You are a helpful, respectful and honest assistant.",
+        "prompt": "Can you write a poem about open source machine learning?"
+    },
+)
+```
+
+Here is the async equivalent of the above:
```

Comment:
I think I'm against using `async` here, given that there's an actual (and different `asyncio`) way of running predictions async in Python land! WDYT?

Reply:
Ah yes, that is a good point. I want some example of `replicate.models.predictions.create` on this page and wasn't sure how to refer to it — async felt like the best option? Open to other ideas on how to word it though

Reply:
@cbh123 Here's how we word the difference on the playground:

> The run() function returns the output directly, which you can then use or pass as the input to another model. If you want to access the full prediction object (not just the output), use the replicate.predictions.create() method instead. This will include the prediction id, status, logs, etc.

Reply:
Look a bit below for the words we already use: https://github.com/replicate/replicate-python?tab=readme-ov-file#run-a-model-in-the-background

Reply:
This might just want to be a part of that section, thinking about it, otherwise this is rather disjointed.

But, gosh, let's fix these APIs as soon as possible. How horrible. I'm tempted to say let's just use this energy to fix the API. Come grab me in Gather if you want to hash out a design.

Reply:
`replicate.predictions.create` should have an overload that takes a `model` kwarg. 

```python
prediction = replicate.predictions.create(model="meta/meta-llama-3-8b-instruct", input={...})
```

The original `models` namespace indirection was a hedge, and I didn't have a chance to circle back to clean things up.

Reply:
PR with that change here: https://github.com/replicate/replicate-python/pull/290

Please take a look and let me know what you think!